addons/portal/static/src/xml/portal_signature.xml

```diff
@@ -1,35 +1,132 @@
 <templates id="template" xml:space="preserve">
-    <t t-name="portal.portal_signature">
-        <form id="o_portal_sign_accept" method="POST">
-            <input type="hidden" name="csrf_token" t-att-value="widget.options.csrf_token"/>
-            <div class="form-group">
-                <label class="col-form-label" for="name">Your Name</label>
-                <input type="text" name="partner_name" id="o_portal_sign_name" class="form-control" t-att-value="widget.options.partnerName"/>
+    <!--
+        This template is used to generate the SVG containing the signature when
+        using mode 'auto'.
+
+        The following variables have to be defined:
+            - width: (number) the width of the resulting image in px
+            - height: (number) the height of the resulting image in px
+            - font: base64 encoded font to use
+            - text: (string) the name to draw
+            - type: (string)
+                -> if set to 'signature', it will adapt the characters width to
+                fit the whole text in the image.
+                -> otherwise, it will adapt the space between characters to fill
+                the image with the text.
+    -->
+    <t t-name="portal.sign_svg_text" name="SVG Signature Text">
```

Comment:
self note: this should be split it another template file, portal_name_and_signature